cpg-core/src/main/java/de/fraunhofer/aisec/cpg/TranslationConfiguration.java

```diff
@@ -140,6 +140,9 @@
    */
   final boolean typeSystemActiveInFrontend;
 
+  /** This is the data structure for storing the compilation database. */
+  final Map<File, List<String>> compilationDatabase;
```

Comment:
Some more specific documentation on what is stored in file and its relation to the list of strings that explains the format, e.g. stores a mapping from files that have to be compiled to their file specific compilation parameters/commands

Reply:
Added documentation for this in the latest commit.

Reply:
@konradweiss is the documentation enough?